### PR TITLE
add registry rules for Api

### PR DESCRIPTION
**Pull request by theganyo**
WIP toward #933

Note: Check framework needs ability to do name lookup (#936) before the issue can be completed. However, this PR works standalone, so if we can get it in, it would be helpful from a dependency standpoint.

### CODE REVIEW

File: cmd/registry/cmd/check/lint/problem.go [hunk 1]
```diff
@@ -26,6 +26,18 @@ const (
 	ERROR
 )
 
+func (s Severity) String() string {
+	switch s {
+	case INFO:
+		return "INFO"
+	case WARNING:
+		return "INFO"
```
Comment (timburks) [inline cmd/registry/cmd/check/lint/problem.go:34]: return "WARNING" ?

File: cmd/registry/cmd/check/rules/rule110/rule110.go [hunk 2]
```diff
@@ -1,18 +1,18 @@
-// Copyright 2022 Google LLC
+// Copyright 2023 Google LLC. All Rights Reserved.
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     https://www.apache.org/licenses/LICENSE-2.0
+//    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (timburks) [inline cmd/registry/cmd/check/rules/rule110/rule110.go:7]: Did you mean to change https to http?

**Review by timburks — APPROVED** [review]

File: cmd/registry/cmd/check/lint/problem.go (hunk 1 above)
Comment (theganyo) [reply cmd/registry/cmd/check/lint/problem.go]: fixed

File: cmd/registry/cmd/check/rules/rule110/rule110.go (hunk 2 above)
Comment (theganyo) [reply cmd/registry/cmd/check/rules/rule110/rule110.go:7]: yes, this was inconsistent with all other notices

**Pull request merged**